compiler/src/iree/compiler/Codegen/Common/TileDispatchUsingInterface.cpp

```diff
@@ -65,6 +65,19 @@ getDistributeLBAndStep(OpBuilder &b, Location loc, OpFoldResult lb,
   return {distributeLB, distributeStep};
 }
 
+// Helper function to change arith.constant to i64 attribute.
+static void changeArithCstToI64Attr(OpBuilder &b,
+                                    SmallVectorImpl<OpFoldResult> &constants) {
+  for (auto &val : constants) {
```

Comment:
nit: Do not use auto when the type is not obvious based on the RHS only.

```suggestion
  for (OpFoldResult &val : constants) {
```